buildenv/jenkins/variables/defaults.yml

```diff
@@ -296,8 +304,8 @@ x86-64_linux:
   build_env:
     cmd:
       17: 'source /home/jenkins/set_gcc7.5.0_env'
-      18: 'source /home/jenkins/set_gcc7.5.0_env'
-      next: 'source /home/jenkins/set_gcc7.5.0_env'
+      18: 'source /home/jenkins/set_gcc_10.3.0_env'
```

Comment:
Does it work to set `source 7.5` for jdk8 and 11? I'd prefer to do xlinux similarly to the other platforms if possible.

Reply:
Adam suggested we could do this, which avoids changes:
```
8: ''
11: ''